plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisRecordCursor.java

```diff
@@ -201,9 +202,9 @@ private void generateRowValues(String keyString, String valueString, @Nullable M
         byte[] keyData = keyString.getBytes(StandardCharsets.UTF_8);
         byte[] stringValueData = valueString.getBytes(StandardCharsets.UTF_8);
         Optional<Map<DecoderColumnHandle, FieldValueProvider>> decodedKey = keyDecoder.decodeRow(keyData);
-        Optional<Map<DecoderColumnHandle, FieldValueProvider>> decodedValue = valueDecoder.decodeRow(
-                stringValueData,
-                hashValueMap);
+        Optional<Map<DecoderColumnHandle, FieldValueProvider>> decodedValue = valueDecoder instanceof RedisRowDecoder
```

Comment:
People will wonder why there is no same logic for `keyDecoder`. Let's leave code comment.

Reply:
I added the code comment.